script_mteb_french/get_model_specs.py

```diff
@@ -0,0 +1,65 @@
+import sys
```

Comment:
Just add a comment at the begining of this script to explain its utility (e.g. """ Script for extracting models' specifications """")

Reply:
Done :)
I also added a #TODO : the method to get the number of parameters doesn't work for all models. It shoud be replace by the method implemented by @wissam-sib (can be done in another issue)